courses/serializers_test.py

```diff
@@ -271,8 +274,19 @@ def test_serialize_program_enrollments(settings, receipts_enabled):
     program = ProgramFactory.create()
     course_run_enrollments = CourseRunEnrollmentFactory.create_batch(
         3,
-        run__course__program=factory.Iterator([program, program, None]),
-        run__course__position_in_program=factory.Iterator([2, 1, None]),
+        run=factory.Iterator(
+            [
+                CourseRunFactory.create(
+                    course=CourseFactory.create(program=program, position_in_program=2)
+                ),
+                CourseRunFactory.create(
+                    course=CourseFactory.create(program=program, position_in_program=1)
+                ),
+                CourseRunFactory.create(
+                    course=CourseFactory.create(program=None, position_in_program=None)
+                ),
+            ]
```

Comment:
Is there any specific reason for this?

Reply:
The CourseRunFactory generates the CourseFactory with `page` as None (required for other tests) as discussed above. So here we need course with `page` not as None. To do so the CourseFactory is called so the `page` element can be created to pass EnrollmentSerializer test.